fall18/p4/README.md

```diff
@@ -1,3 +1,301 @@
 # Project 4
 
-More details coming soon.
+The project focuses on the IMDB movies dataset that contains a list of movies along with information like cast, director, rating, genre, plot description etc. You can open the IMDB-Movie-Data file in Excel or some other software to get an idea of what the dataset contains.
+
+We will be working with **strings**, **loops**, **conditionals** and **command line arguments** in this project.
+
+To start, download the 4 files given below into your project directory.
+
+* [main.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/main.py)
+* [project.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/project.py)
+* [test.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/test.py)
+* [IMDB-Movie-Data.csv](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/IMDB-Movie-Data.csv)
+
+**A reminder:** You should make your changes ONLY to the **main.py** file.
+DO NOT EDIT ANY OTHER FILES.
+
+After downloading all the files into the same directory try running the command below:
+
+```
+python test.py
+```
+
+and you should see the following: 
+```
+To be filled
+
+```
+
+# Introduction to Command Line inputs
+
+Important Note :  In this project we will be working on using commandline arguments to pass inputs to your program. Command-line arguments are specified while running the program file on the commandline. For example consider the following:
+
+```python
+python main.py upper "Steven Spielberg"
+``` 
+
+The above will pass the two arguments `upper` and `"Steven Spielberg"` to main.py via the command line and are called commandline arguments. We have written code inside the main.py file provided to you to read these arguments and provide to you for use in defining the functions for this project. The commandline arguments are called command1 and command2. There are a number of options that can be passed as command1 and command2 and they are described in the directions given below.
```

Comment:
The above will pass the two arguments `upper` and `"Steven Spielberg"` to main.py via the command line and are called commandline arguments. We have written code inside the main.py file provided to you to read these arguments and provide to you for use in defining the functions for this project. 
In the code provided to you, the command line arguments are called command1 and command2. In the above example, command1 contains the string "upper" and command2 contains the string "Steven Spielberg". 
You will be enabling functionality for these command line arguments, so that you can use the functions you write directly from the command line!